docs/architecture/self-hosted/installation-helm/installation-helm.mdx

```diff
@@ -72,14 +75,14 @@ Prophecy is installed via Helm chart called the Prophecy-installer. The followin
 
 Here are some configurations that are commonly changed from the default.
 
-#### Internal repository
-
-During the Helm installation, docker images are pulled from Prophecy's public Google Container Registry `gcr.io/prophecy-share`. If you want to use your own internal container registry, you can specify it by setting the `global.repository` value during installation. Make sure to download the images from the public GCR first.
-
 #### Nginx ingress controller
 
 By default, Prophecy will install its own managed Nginx ingress controller during the Helm installation. If you want to use your own Nginx ingress controller already installed on your Kubernetes cluster, you can do so if it supports external service exposure via a LoadBalancer. You'll have to set `global.disableIngressCreation` to true, and set `global.ingressController.class` to the appropriate name.
 
+#### Internal repository
+
+During the Helm installation, docker images are pulled from Prophecy's public Google Container Registry `gcr.io/prophecy-share`. If you want to use your own internal container registry, you can specify it by setting the `global.repository` value during installation. Make sure to download the images from the public GCR first.
```

Comment:
Instead of this:- Make sure to download the images from the public GCR first.
We can use:- Make sure to push all the images to your private registry first.